### PR TITLE
commandline-usage: separate stand-alone + initial ledger options

### PR DESCRIPTION
**Pull request by intelliot**
Clarify: Initial Ledger Options is not a subcategory of Stand-Alone Mode Options

@thejohnfreeman @mtrippled @pwang200

### CODE REVIEW

File: content/references/http-websocket-apis/commandline-usage.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ Daemon mode is the default mode of operation for `rippled`. In addition to the [
 
 The following option has been removed: `--validateShards`. [Removed in: rippled 1.7.0][]
 
-## Stand-Alone Mode Options
+### Stand-Alone Mode Options
```
Comment (pwang200) [inline content/references/http-websocket-apis/commandline-usage.md]: ```## Daemon Mode Options```
has two ##. Should Stand-Alone Mode Options match that?

Comment (intelliot) [inline content/references/http-websocket-apis/commandline-usage.md]:
```suggestion
## Stand-Alone Mode Options
```

File: content/references/http-websocket-apis/commandline-usage.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ Run in [stand-alone mode](rippled-server-modes.html). In this mode, `rippled` do
 
 ### Initial Ledger Options
```
Comment (intelliot) [inline content/references/http-websocket-apis/commandline-usage.md]:
```suggestion
## Initial Ledger Options
```

File: content/references/http-websocket-apis/commandline-usage.md (hunk 1 above)
Comment (intelliot) [reply content/references/http-websocket-apis/commandline-usage.md]: With 4e625a2, updated so both are at the `##` level

**Comment by mDuo13** [comment]
I'm not convinced that most of these options work meaningfully in networked mode. Most of the time, the server will disregard whatever initial ledger it loaded if its trusted validators are working on something completely different. In what cases are _any_ of these settings useful in networked mode?

**Comment by mtrippled** [comment]
--valid is useful to start the initial validator in a network that needs to be restarted, such as for testing, or something unthinkable.
--ledger can be used in the same type of circumstance to choose a specific ledger.

The logic for the fast_load config option uses the same code path as --load, but reads the disk in parallel.

I don't know if the others are useful in network mode.

**Comment by intelliot** [comment]
note: conversations with the team suggest that these options are not only for standalone mode

**Comment by mDuo13** [comment]
Updated with new information to clarify the uses of these options for starting networks (devnets and sidechains included) as well as debugging.

File: content/references/http-websocket-apis/commandline-usage.md [hunk 3]
```diff
@@ -78,18 +77,19 @@ rippled -a [OPTIONS]
...
-| `--load`              | **DEPRECATED** Intended for debugging. Only load the initial ledger from the ledger store on disk. |
-| `--replay`            | Intended for debugging. Use with `--ledger` to replay a ledger close. Your server must have the ledger in question and its direct ancestor already in the ledger store. Using the previous ledger as a base, the server processes all the transactions in the specified ledger, resulting in a re-creation of the specified ledger. With a debugger, you can add breakpoints to analyze specific transaction processing logic. |
-| `--start`             | Intended for debugging. Start with a new genesis ledger that has all known amendments (except those the server is configured to vote against) enabled. This makes the functionality of those amendments available right away, instead of needing to wait two weeks for the [Amendment Process](amendments.html). |
-| `--valid`             | **DEPRECATED** Intended for debugging. Consider the initial ledger a valid network ledger even before fully syncing with the network. |
+| `--load`              | Use only the ledger store on disk when loading the initial ledger. |
```
Comment (ckeshava) [inline content/references/http-websocket-apis/commandline-usage.md:88]: are we re-enabling the deprecated options (`--load` and `--valid`)?

Comment (intelliot) [reply content/references/http-websocket-apis/commandline-usage.md:88]: yes, my understanding is that those options are no longer considered to be deprecated

**Comment by intelliot** [comment]
FWIW, I approve of the commits added by @mDuo13 above

**Comment by ckeshava** [comment]
I see, okay
Regards,
Keshava

**Comment by mDuo13** [comment]
Link checker failures are the same ones already fixed in #2041, so this is good to go

**Pull request merged**